temp zip

aristanfradejas opened the pull request:
- Login is connected to DB,
- Can only login if in DB
- Will add login attempts counter later  

*still trying to learn how to call user data to other forms, so the greeting label in MainMenu is still LASTNAME, FIRSTNAME, etc.

aristanfradejas committed:
Add files via upload

- Login is connected to DB,
- Can only login if in DB
- Will add login attempts counter later  

*still trying to learn how to call user data to other forms, so the greeting label in MainMenu is still LASTNAME, FIRSTNAME, etc.

Commit:
From Aristan's zip file

Commit:
Deleted the zip file

The contents of the zip file already in the branch.

njraguas reviewed (approved):
No need to put it in a .zip file. Just upload the folder of the updated program in your branch.

After successfully searching the user in the database, throw the values of its corresponding row in the user = new Account() instantiation.
Pero before you do that, you must create the List<Transaction_History> transaction_history muna.

aristanfradejas committed:
Added Login Attempts counter

aristanfradejas committed:
Minor form design fixes

- name label in Main Menu is now centered
- MaxLength of Receiver's Account Number is limited to 8
- MessageBox that pops up when confirming Transfer of Funds now displays the Receiver's Account Number instead of just a #

* will edit Form Designs tomorrow

aristanfradejas committed:
Minor form design fixes and added LoginAttempts counter

- deleted redundant folder
- added LoginAttempts counter in LoginFrm
- name label in Main Menu is now centered
- MaxLength of Receiver's Account Number is limited to 8
- MessageBox that pops up when confirming Transfer of Funds now displays the Receiver's Account Number instead of just a #

* will edit Form Designs tomorrow

aristanfradejas committed:
Major Form Design Overhaul

aristanfradejas committed:
Form Design Overhaul

- removed redundant folder

aristanfradejas committed:
Button Inconsistencies

- fixed button capitalization inconsistencies

Commit:
Fixed the date label in all forms
